main.nf

```diff
@@ -13,64 +13,154 @@ nextflow.enable.dsl = 2
 
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    GENOME PARAMETER VALUES
+    IMPORT FUNCTIONS / MODULES / SUBWORKFLOWS / WORKFLOWS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-// params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
+include { RIBOSEQ                 } from './workflows/riboseq'
+include { PREPARE_GENOME          } from './subworkflows/local/prepare_genome'
+include { PIPELINE_INITIALISATION } from './subworkflows/local/utils_nfcore_riboseq_pipeline'
+include { PIPELINE_COMPLETION     } from './subworkflows/local/utils_nfcore_riboseq_pipeline'
+
+include { getGenomeAttribute      } from './subworkflows/local/utils_nfcore_riboseq_pipeline'
```

Comment:
did you checked that this is working properly with iGenomes references?

Reply:
cf https://github.com/nf-core/rnaseq/pull/1252

Reply:
Seems to work